tools/mkpkg/update_binary-addons

```diff
@@ -89,8 +89,7 @@ validate_pkg_url() {
 }
 
 if [ ! -d $KODI_DIR ] ; then
-  echo "meh.. $KODI_DIR does not exist"
-  exit 127
+  git_clone https://github.com/xbmc/xbmc master $KODI_DIR
```

Comment:
Hmm actually the use of `master` everywhere makes me wonder why we pass a <branch-name> parameter at all.